migrations/20230311081917_envelopes_table.sql

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE IF NOT EXISTS envelopes (
   id              INTEGER PRIMARY KEY AUTOINCREMENT,
+  timestamp       BLOB,
```

Comment:
The `timestamp` name is confusing since it doesn't tell us what this timestamp is referring to, or what this time means/represents.